src/main/java/com/alysaa/geyserupdater/spigot/util/GeyserSpigotDownloader.java

```diff
@@ -28,7 +31,13 @@ public static boolean updateGeyser() {
 
             @Override
             public void run() {
-                String fileUrl = "https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar";
+                String fileUrl = null;
+                try {
+                    fileUrl = "https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/" + GeyserProperties.getGeyserGitPropertiesValueForPropertyKey("git.branch") + "/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar";
+                } catch (IOException e) {
+                    logger.severe("Failed to get the current Geyser build's Git branch!");
+                    e.printStackTrace();
```

Comment:
there should be a 
```
downloadSuccess = false;
return;
```
here so that we don't try downloading with a null fileUrl

Reply:
Addressed in `5b4e9fa7b4aad55af9514accec6a2146d15c96e0`.